tailwind.config.cjs

```diff
@@ -0,0 +1,8 @@
+/** @type {import('tailwindcss').Config} */
+module.exports = {
+  content: ["./src/**/*.{html,js,ts,tsx}"],
```

Comment:
Remove js since you won't use it in this project.

Reply:
Noted, removed.